story/src/2-Spinner.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Spinner } from '../../frontend/src/component/ui/Spinner';
+
+export default {
+  title: 'UI/Spinner',
+  component: Spinner
+};
+
+export const spinner = () => {
+  return <Spinner />;
```

Comment:
Can we wrap the spanner with a background? It's too to see the orbitals and planet on storybook.

Reply:
I'll see what I can do with Storybook Backgrounds. I'll wrap the component with background as fallback in case that doesn't work out.

Reply:
Here's how it looks now, using the dark color that comes with Storybook Backgrounds addon

<img width="1194" alt="image" src="https://user-images.githubusercontent.com/3276350/92201412-78ce5480-ee4a-11ea-8e07-2156381449c5.png">

Reply:
One caveat, if you switch background color, it'll always stay that color, even when selecting `Spinner` story. I'm going to fallback to wrapping.

Reply:
<img width="1675" alt="image" src="https://user-images.githubusercontent.com/3276350/92201788-6acd0380-ee4b-11ea-9a39-24a8a79c021f.png">